src/components/SarStakeWidget/index.tsx

```diff
@@ -104,6 +104,15 @@ export default function SarManageWidget() {
       approval === ApprovalState.PENDING ||
       (approvalSubmitted && approval === ApprovalState.APPROVED));
 
+  const fiatNumberFormmater = new Intl.NumberFormat(undefined, {
+    notation: 'compact',
+    style: 'currency',
+    currency: 'USD',
+    currencySign: 'accounting',
+    currencyDisplay: 'narrowSymbol',
+    maximumFractionDigits: 2,
+  });
```

Comment:
lets use `numeral` as we have been using that at multiple places to be consistent.